CONTRIBUTING.md

```diff
@@ -15,3 +15,24 @@
 - Do not use camel case or underscores for folder names.
 - Arrange your folders in alphabetical order.
 - Folder name or folder description should not contain special characters.
+- 
+## Add an Extension
+
+1. Edit `cmd/populator/modules.go` under `func init() `  with the extension details.
+```
+{
+    name:        "liquibase-commercial-dynamodb",
+    category:    Extension,
+    url:         "https://repo1.maven.org/maven2/org/liquibase/ext/liquibase-commercial-dynamodb",
+    artifactory: Maven{},
+}
+```
+3. Edit `internal/app/packages.json` with the extension details.
+```
+{
+    "name": "extension-name",
+    "category": "extension", // or "driver"
+    "versions": []
+  }
+```
+4. From the Terminal run command : `make generateExtensionPackages`. It will automatically update the `internal/app/packages.json` file with values for `algorithm`, `checksum`, `liquibaseCore`, `path` 
```

Comment:
instead of running this from the terminal could we have it trigger the "update packages" workflow on PR?

Reply:
we can run the workflow from a branch from here https://github.com/liquibase/liquibase-package-manager/actions/workflows/nightly-update-packages.yml to make it happen. Correct me if I am wrong @mcred

Reply:
Adding the following or something similar to `nightly-update-packages` should cover it.
```
on:
  workflow_dispatch:
  schedule:
    - cron: '5 6 * * *'
  push:
    branches:
      - master
    paths:
      - "cmd/populator/**"
```

Reply:
Thanks @mcred 
working as expected.
Example : https://github.com/liquibase/liquibase-package-manager/actions/runs/9976289363/job/27568093384
Automatic PR : https://github.com/liquibase/liquibase-package-manager/pull/404/files